code/modules/spells/ability_types/realized.dm

```diff
@@ -424,8 +689,8 @@
 	if(owner.stat == DEAD)
 		for(var/mob/living/carbon/human/H in GLOB.player_list)
 			if(H.stat != DEAD)
-				H.adjustBruteLoss(-100) // It heals everyone to full
-				H.adjustSanityLoss(-100) // It heals everyone to full
+				H.adjustBruteLoss(-300) // It heals everyone to full
+				H.adjustSanityLoss(-300) // It heals everyone to full
```

Comment:
Unmentioned tripling of the on-death global healing. Also I don't believe it's necessary to buff this as it's a global heal. Though I will say, this buff does not seem to remove itself when the owner is dead.